CHANGELOG.rst

```diff
@@ -15,6 +15,10 @@ Unreleased
 ~~~~~~~~~~
 *
 
+[0.8.5] - 2020-12-24
+~~~~~~~~~~~~~~~~~~~~~
+* Upgrading celery to 5.0.4
+
```

Comment:
Does this need to be deployed?  Generally you don't need a new version number just to upgrade a dependency.

Reply:
We need to release the package to update python classifiers on PyPI as python 3.5 support dropped earlier.